javascript/src/controllers/remote-modal-controller.js

```diff
@@ -50,11 +48,7 @@ export class RemoteModalController extends Controller {
     this.backgroundTarget.addEventListener('click', this._closeModal)
   }
 
-  openModal (content) {
-    this.isWide
-      ? this.wrapperTarget.classList.add('wide')
-      : this.wrapperTarget.classList.remove('wide')
-
+  openModal(content) {
```

Comment:
esto es lo que ha de haber tronado en StandardJS

```suggestion
  openModal (content) {
```

Reply:
Ya lo arregle